.eslintrc

```diff
@@ -0,0 +1,20 @@
+{
+  "env": {
+    "node": true
+  },
+  "rules": {
+    "no-bitwise": 2,
+    "camelcase": 0,
+    "curly": 2,
+    "eqeqeq": 2,
+    "wrap-iife": [2, "any"],
+    "indent": [2, 4, { "SwitchCase": 1 }],
+    "no-use-before-define": [2, "nofunc"],
+    "new-cap": 2,
+    "no-caller": 2,
+    "quotes": [2, "single"],
+    "no-undef": 2,
+    "no-unused-vars": 2,
+    "strict": 0
+  }
+}
```

Comment:
you might want to look at https://www.npmjs.com/package/semistandard (standardjs.com). it's pretty much very close to this eslint config + more.